test/e2e/kubectl.go

```diff
@@ -238,6 +240,48 @@ var _ = Describe("Kubectl client", func() {
 		})
 	})
 
+	Describe("Kubectl run", func() {
+		var nsFlag string
+		var rcName string
+
+		BeforeEach(func() {
+			nsFlag = fmt.Sprintf("--namespace=%v", ns)
+			rcName = "e2e-test-nginx-rc"
+		})
+
+		AfterEach(func() {
+			runKubectl("stop", "rc", rcName, nsFlag)
+		})
+
+		It("should create an rc from an image", func() {
+			image := "nginx"
+
+			By("running the image " + image)
+			runKubectl("run", rcName, "--image="+image, nsFlag)
+			By("verifying the rc " + rcName + " was created")
+			rc, err := c.ReplicationControllers(ns).Get(rcName)
+			if err != nil {
+				Failf("Failed getting rc %s: %v", rcName, err)
+			}
+			containers := rc.Spec.Template.Spec.Containers
+			if containers == nil || len(containers) != 1 || containers[0].Image != image {
+				Failf("Failed creating rc %s for 1 pod with expected image %s", rcName, image)
+			}
+
+			By("verifying the pod controlled by rc " + rcName + " was created")
+			label := labels.SelectorFromSet(labels.Set(map[string]string{"run": rcName}))
+			podlist, err := c.Pods(ns).List(label, fields.Everything())
+			if err != nil {
+				Failf("Failed getting pod controlled by rc %s: %v", rcName, err)
+			}
+			pods := podlist.Items
+			if pods == nil || len(pods) != 1 || len(pods[0].Spec.Containers) == 0 || pods[0].Spec.Containers[0].Image != image {
+				runKubectl("get", "pods", "-L", "run", nsFlag)
+				Failf("Failed creating 1 pod with expected image %s. Number of pods = %v", image, len(pods))
```

Comment:
This test always fails here in Jenkins, but not on my machine. 
`Failed creating 1 pod with expected image nginx. Number of pods = 0`

So I tried to debug by adding this kubectl command
`get pods -L run --namespace=e2e-tests-kubectl-eu42p`
```
INFO: 
NAME                      READY     STATUS    RESTARTS   AGE       RUN
e2e-test-nginx-rc-wgnvd   0/1       Pending   0          1s        e2e-test-nginx-rc
```

And looks like it's actually working. Can't figure out why this test fails... is it because the pod status is `Pending`? (On my machine it shows `Running`)